src/scripts.js

```diff
@@ -21,6 +25,39 @@ let hydrationRepo;
 let sleepRepo;
 let activityRepo;
 
+const fetchedUserData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData')
```

Comment:
Nice user of fetch to get data from the server!

Reply:
Log gone!